src/main.ts

```diff
@@ -40,11 +40,18 @@ async function getRepoFlake(): Promise<string> {
   return `builtins.getFlake("${flakeUrl}")`;
 }
 
-function maybeAddNixpkgs(pkg: string) {
-  if (pkg.indexOf("#") < 0) {
-    return "nixpkgs#" + pkg;
-  } else {
+async function maybeAddNixpkgs(pkg: string): Promise<string> {
+  const res = await getExecOutput("nix", ["flake", "metadata", pkg], {
```

Comment:
Thinking out loud: This could be moved to `nix.ts` (and `runCmd` in `nix.ts` probably simplified with `getExecOutput` :see_no_evil:)

Reply:
Done!